aio/aio-proxy/aio_proxy/response/formatters/etablissements.py

```diff
@@ -102,6 +102,10 @@ def format_siege(siege=None, is_non_diffusible=False):
 
 
 def hide_non_diffusible_etablissement_fields(etablissement):
+    # in order to keep `liste_enseignes` as an array of "NON-DIFFUSIBLE" strings
+    etablissement["liste_enseignes"] = ["[NON-DIFFUSIBLE]"] * len(
+        etablissement["liste_enseignes"]
```

Comment:
Not an expert in python, but this read like it s gonne return : 
["[NON-DIFFUSIBLE]"], ["[NON-DIFFUSIBLE]"], ["[NON-DIFFUSIBLE]"]

for an etablissement with three enseignes ?